sdk/core/Azure.Core/tests/TransportFunctionalTests.cs

```diff
@@ -239,7 +239,7 @@ public async Task CanGetAndSetMethod(RequestMethod method, string expectedMethod
          [TestCaseSource(nameof(AllHeadersWithValuesAndType))]
          public async Task CanGetAndAddRequestHeaders(string headerName, string headerValue, bool contentHeader)
          {
-             StringValues httpHeaderValues;
+            Microsoft.Extensions.Primitives.StringValues httpHeaderValues;
```

Comment:
Do we still need this qualification since we don't carry own `StringValues ` anymore?